### PR TITLE
Develop

### LINKED ISSUE #2: Exercise
Wczytaj od użytkownika dowolną liczbę całkowitą różną od 0. Jeżeli liczba jest większa od 0 wyświetl na ekranie wszystkie liczby z przedziału [0, N], gdzie N jest liczbą podaną przez użytkownika z odstępem co 0.1. Jeśli liczba jest mniejsza od 0 wyświetl liczby od 0 do N (w porządku malejącym).

### PR DESCRIPTION
**Pull request by tstepnik**
Closes #2

### CODE REVIEW

**Comment by rbobko93** [comment]
Make PR (pull request) name more informational

File: src/TaskNumbers.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+
+public class TaskNumbers {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        double x = 0;
```
Comment (rbobko93) [inline src/TaskNumbers.java]: It was supposed to be int.

File: src/TaskNumbers.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        System.out.println("Podaj liczbę:");
+        double usersNumber = sc.nextDouble();
```
Comment (rbobko93) [inline src/TaskNumbers.java]: Read int

Comment (rbobko93) [reply src/TaskNumbers.java]: Then convert it to double

Comment (rbobko93) [reply src/TaskNumbers.java]: Change variable name to userNumber

File: src/TaskNumbers.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+        if (usersNumber > 0) {
+            double xx = 0;
```
Comment (rbobko93) [inline src/TaskNumbers.java]: This var is never used.

File: src/TaskNumbers.java (hunk 1 above)
Comment (rbobko93) [reply src/TaskNumbers.java]: Leave it a double. My bad. Move it to if body though

File: src/TaskNumbers.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+            while (x < usersNumber + 0.1) {
+                System.out.printf("%.1f" + " ", x);
+                x += 0.1;
+            }
+        } else if (usersNumber < 0) {
+            double y = 0;
+            double yy = 0;
```
Comment (rbobko93) [inline src/TaskNumbers.java]: Never used

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Remove folder out/ and .iml files from the project. Add them to gitignore.

**Review by rbobko93 — CHANGES_REQUESTED** [review]
Remove /out and .iml

**Comment by tstepnik** [comment]
> Remove /out and .iml

Files removed.

**Review by rbobko93 — APPROVED** [review]

**Pull request merged**